app/helpers/textual_helper.rb

```diff
@@ -0,0 +1,81 @@
+module TextualHelper
```

Comment:
Minor, but can we name this `TextualSummaryHelper`.  The "Summary" part is actually the important part as it refers to the summary screens.  In the past we had 2 kinds of summary screens, textual and graphical.  I believe the graphical has since been completely removed.  So, maybe now's the time to just call it `SummaryHelper`